mandatory/1-fix-functions.js

```diff
@@ -10,31 +10,27 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(isHappy) {
+  
+  if (isHappy === true) {
```

Comment:
So this line was actually fine as was -- `isHappy` can have two values `true` or `false` and that's all that's needed for this if statement. Basically, if we replace `isHappy` with its value, you get:
```javascript
true === true // evaluates to true
false === true // evaluates to false
```
Of course, this assumes only a boolean would ever be passed to the function.